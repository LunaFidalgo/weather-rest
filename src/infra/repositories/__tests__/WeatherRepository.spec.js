const WeatherRepository = require('../WeatherRepository')
const weatherClient = require('./__mocks__/weatherClient')

describe('Weather repository', () => {
  let repo

  beforeEach(() => {
    repo = new WeatherRepository({ weatherClient })
  })

  describe('Query data', () => {
    test('Get weather info', async () => {
      const weather = repo.getWeatherInfo()
      expect(weather).toMatchSnapshot()
    })
  })
})
