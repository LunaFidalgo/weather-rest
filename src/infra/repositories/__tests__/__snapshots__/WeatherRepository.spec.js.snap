// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Weather repository Query data Get weather info 1`] = `Promise {}`;
